lib/options.go

```diff
@@ -406,7 +407,9 @@ func (o Options) Apply(opts Options) Options {
 	if opts.NoSetup.Valid {
 		o.NoSetup = opts.NoSetup
 	}
-	if opts.SetupTimeout.Valid {
+	if opts.SetupTimeout.Valid && opts.SetupTimeout.Duration <= 0 {
+		return o, errors.New("setupTimeout must be positive")
+	} else if opts.SetupTimeout.Valid {
```

Comment:
I really don't think changing Apply signature is warranted. 

We do validation checks for most of the options, and they all happen in the respected places instead of in Apply - so please do not change the signature of Apply - that will also make the change a lot smaller

Reply:
@mstoykov 
Thank you for comment.

I updated to move in Validate method.
Is it reasonable?